gfx/UltimateCataclysm/pngs_normal_32x32/items/joints/joints.json

```diff
@@ -0,0 +1,17 @@
+[
+  {
+    "id": "joint",
+    "fg": [ "joint" ],
+    "bg": [  ]
+  },
+  {
+    "id": "joint_lit",
+    "fg": [ "joint_lit" ],
+    "bg": [  ]
+  },
+  {
+    "id": "joint_roach",
+    "fg": [ "joint_roach" ],
```

Comment:
Image name is `joint_Roach.png`. I don't know if the build script is case sensitive: if it's not, discard that; otherwise it need to match the file name.